desktop/hardened-desktop/enhanced-container-isolation/faq.md

```diff
@@ -0,0 +1,52 @@
+---
+title: FAQs and known issues
+description: FAQ for Enhanced Container Isolation 
+keywords: enhanced container isolation, security, faq, sysbox
+toc_max: 2
+---
+
+<ul class="nav nav-tabs">
+  <li class="active"><a data-toggle="tab" data-target="#tab3">FAQs</a></li>
+  <li><a data-toggle="tab" data-target="#tab4">Known issues</a></li>
+</ul>
+<div class="tab-content">
+<div id="tab3" class="tab-pane fade in active" markdown="1">
+
+### With Enhanced Container Isolation enabled, can the user still override the `--runtime` flag from the CLI ?
+
+No. With Hardened Desktop enabled, Sysbox is locked as the default (and only) runtime. If a user attempts to override the runtime by launching a container with the standard `runc` runtime, for example `docker run --runtime=runc`, container creation fails. 
```

Comment:
There's a correction needed here:

If a user attempts to override the runtime by launching a container with the standard `runc` runtime, for example `docker run --runtime=runc`, this request is ignored and the container is created through the Sysbox runtime.